core/src/main/kotlin/run/qontract/core/git/SystemGit.kt

```diff
@@ -49,6 +49,11 @@ class SystemGit(private val workingDirectory: String = ".", private val prefix:
     }
 
     override fun inGitRootOf(contractPath: String): GitCommand = SystemGit(File(contractPath).parentFile.absolutePath)
+
+    override fun repoName(): String {
```

Comment:
Might be better to do this using the directory instead of the url. Theoretically the file path can be different from the url, and the file path I believe is also easy to get at. I would like to get your thoughts on this.

Reply:
removed reponame, making use of gitRoot now.